src/tiled/documentmanager.h

```diff
@@ -196,6 +196,7 @@ public slots:
     void hideChangedWarning();
 
     void tilesetImagesChanged(Tileset *tileset);
+    void objectTemplateChanged(ObjectTemplate *objectTemplate);
```

Comment:
This function should not be necessary, since each `MapDocument` already connects to `TemplateManager::objectTemplateChanged`.